CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.1)
 project(libsamplerate VERSION 0.1.9 LANGUAGES C)
 
+option(LIBSAMPLERATE_TESTS ON "Enable to generate test targets")
```

Comment:
setting this option is wrong - it must be changed to 

`option(LIBSAMPLERATE_TESTS "Enable to generate test targets" ON)`

see https://cmake.org/cmake/help/latest/command/option.html

Reply:
Indeed, thanks!